gbfs-validator/cli.js

```diff
@@ -1,17 +1,65 @@
 const { inspect } = require('util')
+const commander = require('commander')
+const fs = require('fs')
+const fsPath = require('path')
 const GBFS = require('./')
+const pjson = require('./package.json');
 
-if (!process.argv[2]) {
-  console.error('Usage: gbfs-validator GBFS_URL')
-  process.exit(1)
+getFeedValidationReport = async (url) => {
+  const gbfs = new GBFS(url)
+  return gbfs.validation()
 }
 
-const gbfs = new GBFS(process.argv[2])
+parseOptions = () => {
+  commander
+    .version(pjson.version, '-v, --version')
+    .usage('[OPTIONS]...')
+    .option('-u, --url <dataset_folder>', 'URL of the GBFS feed')
```

Comment:
Would `dataset_url` be more intuitive here?

```suggestion
    .option('-u, --url <dataset_url>', 'URL of the GBFS feed')
```

Reply:
Done!